### PR TITLE
Erc20 lesson update 19nov

### PR DESCRIPTION
**Pull request by elPiablo**
### PR aims to get *Create your first ERC-20 with Foundry* lesson fully ready for production in 5 to 6 stages/commits

- [x] 1. OpenZeppelin have updated their releases to solc version 0.8.20 with breaking changes. Our first commit will: 

update solidity pragma statements; redeploy contact; update terminal output code; renew Etherscan screenshots (public/assets/lessons) (see commits: https://github.com/Developer-DAO/academy/pull/268/commits/218e0585f8eb29b5b65dee8915913a6a4f1f7f34 and https://github.com/Developer-DAO/academy/pull/268/commits/310cdae722d118c2fca91ec82cbd071605ed6552)

- [x] 2. add 1/4 and 2/4 of checkpoint questions (see commit: https://github.com/Developer-DAO/academy/pull/268/commits/c2d7cc51fe71b09445707b497642d58e861d5783) 

- [x] 3. add 3/4 and 4/4 of checkpoint questions (see commit: https://github.com/Developer-DAO/academy/pull/268/commits/0b79decb6a61a5dfd9abdef4e687744ff0365039 )

- [x] 4. add final quiz (see commit: https://github.com/Developer-DAO/academy/pull/268/commits/21b61b41bee1cda2230710109e3d14b2041a385d)

- [x] 5. add outro final paragraph for learners' next possible steps including other tracks and https://handbook.developerdao.com/ (see commit: )

~~- [ ] 6. Need to get info on POKT (Nodies and Grove) - there doesn't seem to be a shared public URL for an endpoint. And need to get in touch with Zerion to give us a little guide on what it is and how to set up. (see issue: https://github.com/Developer-DAO/academy/issues/269#issue-2000993140)~~ we have an open issue for this, so we can ignore it in the context of this PR

- [x] extra: add SEO friendly title and description (see commit: https://github.com/Developer-DAO/academy/pull/268/commits/7773888965ba3d3cf73523e473e2a639838efd9b)

### CODE REVIEW

File: src/utils/quizzes/quiz-lesson-6.json [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+          "answer": "Windows"
+        },
+        {
+          "answer": "Unix based systems",
+          "correct": true
+        }
+      ]
+    },
+    {
+      "question": "How many methods are required by the ERC-20 standard to achieve the necessary functionality for a token contract?",
```
[the remark this answers is not in the document]
Comment (wolovim) [reply src/utils/quizzes/quiz-lesson-6.json:36]: @elPiablo this is a break from the current policy to give this level of repo permissions to Core Team members. not strictly opposed to adjusting that, but worth a quick convo in discord. if we don't grant further permissions, one alternative is to merge your work, the happy can follow up with a quick PR of his own, from his fork of the codebase.

[the remark this answers is not in the document]
Comment (elPiablo) [reply src/utils/quizzes/quiz-lesson-6.json:36]: Hey @wolovim 
I have six tasks on the list here, but we can settle for the first five. Item number 6 is a generic question about partners we need to answer at some stage ASAP, but this tutorial is not dependent on it to be merged (original PR message for clarity => https://github.com/Developer-DAO/academy/pull/268#issue-2000991723)
You have quite a bit to get here, but I planned the commits before starting on them, so I hope you experience them cleanly and succinctly. 
Have fun! 😄

[the remark this answers is not in the document]
Comment (elPiablo) [reply src/utils/quizzes/quiz-lesson-6.json:36]: > @elPiablo this is a break from the current policy to give this level of repo permissions to Core Team members. not strictly opposed to adjusting that, but worth a quick convo in discord. if we don't grant further permissions, one alternative is to merge your work, then happy can follow up with a quick PR of his own, from his fork of the codebase.

yeah, I can shoot Happy a message and advise that course of action :)

**Comment by elPiablo** [comment]
@wolovim just added an SEO friendly title and description that I had forgotten :https://github.com/Developer-DAO/academy/pull/268/commits/7773888965ba3d3cf73523e473e2a639838efd9b

**Review by wolovim — APPROVED** [review]
upon skim, looks to be a solid iteration. approving without fine-toothed combing, so happy to make his edits while its still on staging.

**Comment by elPiablo** [comment]
@Markkos89 can you debug the hydration errors on this branch, please? Cheers!

**Comment by Markkos89** [comment]
> @Markkos89 can you debug the hydration errors on this branch, please? Cheers!

Note: with piablo saw that removing some comments that were invalidating some eslint the hydration error is not being triggered anymore

**Pull request merged**